charts/seed-bootstrap-crds/charts/vpa/templates/verticalpodautoscalercheckpoints.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or .Values.admissionController.enabled .Values.recommender.enabled .Values.updater.enabled }}
+{{- if or .Values.global.vpaEnabled .Values.enabled }}
```

Comment:
Those CRDs are installed also in the shoots
- `global.vpaEnabled` is used when it is needed in the shoot cluster
- `enabled` is used when it is needed in the seed cluster